RxExample/RxExample/Examples/WikipediaImageSearch/Views/WikipediaSearchViewController.swift

```diff
@@ -13,15 +13,10 @@ import RxCocoa
 #endif
 
 class WikipediaSearchViewController: ViewController {
+    private let searchController = UISearchController(searchResultsController: UITableViewController())
+    
+    @IBOutlet weak var searchBarContainer: UIView!
     
-    private var resultsTableView: UITableView {
```

Comment:
The main reason why I suggest properties is readability. We repeat `resultController.tableView!` in a lot of places instead of just `resultsTableView` :)

Repeating `!` throughout the code also pokes my eyes :)

I'm fine with adding additional commits.